src/main/java/com/woowacourse/matzip/application/RestaurantService.java

```diff
@@ -30,9 +32,23 @@ public List<RestaurantTitleResponse> findByCampusIdOrderByIdDesc(final Long camp
                 .collect(Collectors.toList());
     }
 
-    private RestaurantTitleResponse toResponse(Restaurant restaurant) {
-        double rating = reviewRepository.findAverageRatingByRestaurantId(restaurant.getId())
+    public List<RestaurantTitleResponse> findRandomByCampusId(final Long campusId, final int size) {
```

Comment:
```suggestion
    public List<RestaurantTitleResponse> findRandomsByCampusId(final Long campusId, final int size) {
```
복수 취급이 조금 더 맞을 것 같아요 :)